config/default/crd/bases/infrastructure.cluster.x-k8s.io_vsphereclusters.yaml

```diff
@@ -321,7 +321,6 @@ spec:
                     description: 'UID of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#uids'
                     type: string
                 type: object
-                x-kubernetes-map-type: atomic
```

Comment:
I think this should not have happened 🤔  may be related to #1928

Reply:
Yeah, @srm09 pointed me to some issue on controller-gen IIRC, it keeps changing :)